api.py

```
@@ -1,15 +1,16 @@
 from fastapi.templating import Jinja2Templates
 from fastapi.security.api_key import APIKeyCookie
 from sqlalchemy.orm import Session
 from sqlalchemy import select, delete
 from typing import Union, Annotated
 from starlette.status import HTTP_500_INTERNAL_SERVER_ERROR
-from db import ClubSwimmer, get_db
+from db import ClubSwimmer, ClubSwimmerPb, get_db
 from admin import verify_token
 from scraper import swimrankings
 from scraper.swimrankings import SwimrankingsScraper
+from datetime import time, date, datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)